code/game/objects/items/crayons.dm

```diff
@@ -378,6 +376,11 @@
 		if(!do_after(user, 50, target = target))
 			return
 
+	var/charges_used = use_charges(user, cost)
+	if(!charges_used)
+		return
+	. = charges_used
```

Comment:
Does moving this assignment effect anything? How is this return value used?

Reply:
The return type isn't utilized as far as I can see, but moving the assignment appears to have been negligible outside of the order of operations being corrected..